actix-tls/src/connect/ssl/mod.rs

```diff
@@ -5,3 +5,6 @@ pub mod openssl;
 
 #[cfg(feature = "rustls")]
 pub mod rustls;
+
+#[cfg(feature = "native-tls")]
+pub mod nativetls;
```

Comment:
nit but the mod name should match the crate name like `native_tls`

Reply:
yea. I was match the naming with accept module. Should this be a break change PR and re-name that too?

Reply:
sure